cmd/dra-example-kubeletplugin/state.go

```diff
@@ -157,11 +167,31 @@ func (s *DeviceState) prepareDevices(claim *resourceapi.ResourceClaim) (Prepared
 			return nil, fmt.Errorf("requested GPU is not allocatable: %v", result.Device)
 		}
 
-		device := &drapbv1.Device{
-			RequestNames: []string{result.Request},
-			PoolName:     result.Pool,
-			DeviceName:   result.Device,
-			CDIDeviceIDs: s.cdi.GetClaimDevices([]string{result.Device}),
+		config, err := GetMergedOpaqueDeviceConfig[configapi.GpuConfig](
+			configapi.Decoder,
+			DriverName,
+			result.Request,
+			claim,
+			classes,
+		)
+		if err != nil {
+			return nil, fmt.Errorf("error getting GPU configs: %v", err)
+		}
+
+		// Apply any requested configuration here.
+		//
+		// In this example driver there is nothing to do at this point, but a
+		// real driver would likely need to do some sort of hardware
+		// configuration here, based on the config that has been passed in.
```

Comment:
This is clearer than the previous single line comment. Does it make sense to have an empty stub function that we call here instead? We could then properly document this stub to make it clear that this is something that should be extended in real-world cases.

Reply:
Will add in next iteration.

Reply:
Updated.